sys/include/net/dhcpv6/client.h

```diff
@@ -170,6 +199,44 @@ void dhcpv6_client_conf_prefix(unsigned netif, const ipv6_addr_t *pfx,
                                unsigned pfx_len, uint32_t valid,
                                uint32_t pref);
 
+/**
+ * @brief   Checks if the given network interface is configured
+ *          to use DHCPv6 IA_NA
+ *
+ * @param[in] netif     Network interface to check.
+ *
+ * @return  true, if the network interface is set up for IA_NA.
+ */
+bool dhcpv6_client_check_ia_na(unsigned netif);
+
+/**
+ * @brief   Configures a address lease that is provided by the server.
+ *
+ * @param[in] netif     Network interface the address was for.
+ * @param[in] addr      The assigned address.
+ *
+ * @return  sizeof(ipv6_addr_t) on success.
+ */
+int dhcpv6_client_add_addr(unsigned netif, ipv6_addr_t *addr);
+
+/**
+ * @brief   Deprecates an existing address from an address lease.
+ *
+ * @param[in] netif     Network interface the address was for.
+ * @param[in] addr      The address to deprecate.
+ */
+void dhcpv6_client_deprecate_addr(unsigned netif, const ipv6_addr_t *addr);
+
+/**
+ * @brief   Removes an existing address that originated from an address lease.
+ *
+ * @param[in] netif     Network interface the address was for.
+ * @param[in] addr      The address to remove.
+ *
+ * @return  sizeof(ipv6_addr_t) on success.
```

Comment:
See https://github.com/RIOT-OS/RIOT/pull/16228#discussion_r686142155:

```suggestion
 * @return <0 on error.
```

Reply:
But also consider https://github.com/RIOT-OS/RIOT/pull/16228#discussion_r686161168